doc/nrf/releases_and_maturity/migration/nRF54H20_migration_2.7/migration_guide_2.4.99-cs3_to_2.7_application.rst

```diff
@@ -74,6 +74,38 @@ General
 
      The previously used SOC1-based board files have been removed.
 
+nRF Desktop
+-----------
+
+The following changes have to be made for :ref:`nrf_desktop` application to work in the same way as in the release v2.4.99-cs3.
+
+* Due to the migration to the new hardware model, commonly referred as "hardware model v2", the board target of the nRF54H20 DK has changed to the ``nrf54h20dk/nrf54h20/cpuapp``.
+  Building command for application has changed and it is now:
+
+  .. code-block:: console
+
+    west build -b nrf54h20dk/nrf54h20/cpuapp
+
+* Radio core now uses the :ref:`ipc_radio` application from ``sdk-nrf`` instead of the Bluetooth HCI RPMsg sample from ``sdk-zephyr``.
+* Radio core image configuration files have been moved from :file:`configuration/nrf54h20dk_nrf54h20_cpuapp/child_image/hci_rpmsg` to :file:`configuration/nrf54h20dk_nrf54h20_cpurad/images/ipc_radio` directory.
+* Due to transition to sysbuild, configuration enabling radio core image has been moved from the main application image configuration to the sysbuild configuration.
+  See the following sysbuild Kconfig options related to the radio core image configuration:
+
+    * ``SB_CONFIG_NRF_DEFAULT_IPC_RADIO``
+    * ``SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC``
+
+* Application uses the default memory map based on Zephyr board files and no longer defines an application-specific memory map.
+* Module :file:`dfu_mcumgr_suit.c` has been merged with :file:`dfu_mcumgr.c`.
+  The ``CONFIG_DESKTOP_DFU_MCUMGR_SUIT_ENABLE`` Kconfig option had been removed and replaced by :ref:`CONFIG_DESKTOP_DFU_BACKEND_SUIT <config_desktop_app_options>`.
+* The :file:`prj_suit_smp.conf` configuration for nRF54H20 DK has been removed.
+  Now DFU over SMP is enabled by default in the ``debug`` and ``release`` configurations.
+* Moved to using USB-next stack for USB support.
+  New kconfig option :ref:`CONFIG_DESKTOP_USB_STACK_NEXT <config_desktop_app_options>` has been added to enable it in nRF Desktop application.
+  An USB HID-class instance is now configured through a separate DTS node compatible with ``zephyr,hid-device``.
+  See :ref:`nrf_desktop_usb_state` documentation for details related to USB-next stack integration.
+* Align FLASH writes in the :ref:`nrf_desktop_dfu` to FLASH write block size of the non-volatile memory.
+  It's needed as :ref:`CONFIG_SOC_FLASH_NRF_MRAM_ONE_BYTE_WRITE_ACCESS <config_desktop_app_options>` is no longer available.
```

Comment:
```suggestion
  This is needed because the :ref:`CONFIG_SOC_FLASH_NRF_MRAM_ONE_BYTE_WRITE_ACCESS <config_desktop_app_options>` Kconfig option is no longer available.
```